packages/rancher-istio-1.8/charts/templates/istio-psp.yaml

```diff
@@ -0,0 +1,81 @@
+{{- if .Values.global.rbac.pspEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: istio-psp
+  namespace: {{ template "istio.namespace" . }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: istio-psp
+subjects:
+  - kind: ServiceAccount
+    name: istio-egressgateway-service-account
```

Comment:
Do all of these deployments from istio already run as non-root ? If not that would violate this policy.

Reply:
All of the deployments currently run as non-root. There is a global setting in the istio operator to change `runAsNonRoot` but we do not currently set it to false.